packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -272,6 +274,14 @@ export const getNextResizeGroupStateProvider = (measurementCache = getMeasuremen
   };
 };
 
+// Provides a context property that (if true) tells any child components that
+// they are only being used for measurement purposes and will not be visible.
+const MeasuredContext = provideContext({
```

Comment:
This is probably worth documenting a bit more. Either with more comments or an explicit use case documented on the component's demo page as an example.

Reply:
Is there something specific that you don't understand? Do you understand what ResizeGroup does?

It essentially renders everything twice side-by-side. There is one visible version and hidden version that is for measurement purposes. This context property can be used to know which version a component is being rendered in.

Reply:
Yes, I wrote a lot of the code for ResizeGroup. I want to make sure someone understands why this would be used

Reply:
Oh, I didn't realize that. Then what more do you think needs to be said?